pymc/tests/distributions/test_continuous.py

```diff
@@ -2281,5 +2288,25 @@ def test_normal_icdf(self, dist_params, obs, size):
         dist_params = dict(zip(dist_params_at, dist_params))
 
         x = Normal.dist(*dist_params_at, size=size_at)
-
         scipy_logprob_tester(x, obs, dist_params, test_fn=st.norm.ppf, test="icdf")
+
+    def test_uniform_icdf(self, dist_params, obs, size):
```

Comment:
You need to parametrize the test parameters, check the use of `pytest.parametrize` in the normal test above